Sources/Datadog/RUM/RUMMonitor/Scopes/RUMSessionScope.swift

```diff
@@ -207,21 +205,30 @@ internal class RUMSessionScope: RUMScope, RUMContextProvider {
         )
     }
 
-    private func startApplicationLaunchView(on command: RUMCommand, context: DatadogContext) {
+    internal func startApplicationLaunchView(context: DatadogContext, writer: Writer) {
```

Comment:
suggestion/ So far the RUMScopes architecture was only allowing side-effects that happen in response to `RUMCommand` processing. Everything was implicit and the only internal API on scopes was:
```swift
func process(command: RUMCommand, context: DatadogContext, writer: Writer) -> Bool
```
Here we seem to break this convention and add a precedence: the "start app launch view" routine becomes explicit and can be called from outside, causing session scope to call arbitrary API on view scope. This adds extra complexity to an already complicated RUM architecture - to fully test it now, we'd need to cover all extra usages, like calling `startApplicationLaunchView()` in already running RUM session.

Could we consider adopting this change without introducing new internal APIs in scopes? At the surface, it looks doable, because `RUMSessionScope` has `isInitialSession` flag and `RUMViewScope` has its `isInitialView` attribute. It looks that passing necessary timings and `!= .background` state to scopes `.init()` could be enough.

Alternatively, maybe introducing `RUMStartApplicationLaunchView` command could make RUM code simpler? Such command could be sent once from `RUMMonitor` (right after initialising scopes). It shouldn't require introducing much new pieces to RUM scopes as well. Also, because it will be `RUMMonitor` who emits that command, this won't require such strong surgery in RUM scopes unit tests like we had to do in this PR - the new logic would be additive.

I wonder what you think @maxep @fuzzybinary ?